core/packages/test/scripts/build-binary.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -eu
+
+source scripts/set-env.sh
+
+build_relaychain()
+{
+    if [ ! -d "$relaychain_dir" ] ; then
+        git clone https://github.com/paritytech/polkadot.git $relaychain_dir
+    fi
+    pushd $relaychain_dir
+    git fetch --tags
+    git checkout $relaychain_version
```

Comment:
Fetching tags and checking out branches should be inside the `if` after line 9. We should only do this on the first time initialising the parachain repo. This way we can re-run build_relaychain if we have made modifications to the polkadot repo without the `git checkout` failing.